mmpose/datasets/transforms/formatting.py

```diff
@@ -146,6 +148,13 @@ def transform(self, results: dict) -> dict:
                 gt_fields.set_field(results[key], packed_key)
         data_sample.gt_fields = gt_fields.to_tensor()
 
+        # pack multilevel fields
```

Comment:
I think we better use `gt_fields` and `pred_fields` for both single-level and mulit-level cases, and choose its data type between `PixelData` and `MultilevelPixelData` according to the data (np.ndarray or list) to pack.

Reply:
Agreed. I have modified it.